draft-ietf-quic-multipath.md

```diff
@@ -1239,6 +1239,12 @@ An acknowledgement receiver
 can, however, remember the path over which the PATH_ACK that produced
 the minimum RTT was received, and restart the minimum RTT computation
 if that acknowledgement path changes or is abandoned.
+If acknowledgements are not send consistently over one path, the
+acknowledgement receiver can monitor over with path acknowledgement
+are received any ony use samples for acknowledgements received on the same
+path than the data was sent, if any, or simply use only samples from the
+acknowledgement path with the largest delay.
```

Comment:
Please delete the text after "if any", up to "largest delay." It is hard to find a theoretical justification for using "largest delay".

Reply:
Okay, will remove.